### PR TITLE
229 mock up maatregelen and vereisten in task registry connected to profile based on labels

### PR DESCRIPTION
**Pull request by ChristopherSpelt**
# Description

Describe in detail the changes you are proposing, and the rationale.

Link all GitHub issues fixed by this PR.
Make sure to first open an issue, get community approval and only then create Pull Request to resolve it.
All Pull Requests must have an issue attached to them

Resolves #

## Checklist

Please check all the boxes that apply to this pull request using "x":

-   [x] I have tested the changes locally and verified that they work as expected.
-   [x] I have followed the project's coding conventions and style guidelines.
-   [x] I have rebased my branch onto the latest commit of the main branch.
-   [x] I have squashed or reorganized my commits into logical units.
-   [x] I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

### CODE REVIEW

File: amt/models/project.py [hunk 1]
```diff
@@ -47,6 +47,9 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:  # noqa: ANN401
 
     @property
     def system_card(self) -> ProjectSystemCard:
+        if not hasattr(self, "_system_card"):
+            self._system_card: ProjectSystemCard | None = None
```
Comment (anneschuth) [inline amt/models/project.py:51]: Thanks!

File: amt/api/routes/project.py [hunk 2]
```diff
@@ -244,12 +268,23 @@ async def get_system_card_requirements(
         request,
     )
 
+    # TODO: This is only for the demo of 18 Oct. In reality one would load the requirements from the requirement
```
Comment (anneschuth) [inline amt/api/routes/project.py:271]: is this TODO still relevant?

File: amt/schema/system_card.py [hunk 3]
```diff
@@ -19,4 +21,6 @@ class SystemCard(BaseModel):
     labels: list[dict[str, Any]] = Field(default=[])
     status: str = Field(default=None)
     instruments: list[InstrumentBase] = Field(default=[])
+    requirements: list[RequirementTask] = Field(default=[])
+    measures: list[MeasureTask] = Field(default=[])
```
Comment (anneschuth) [inline amt/schema/system_card.py:25]: did we update the schema as well?

File: amt/api/routes/project.py (hunk 2 above)
Comment (laurensWe) [reply amt/api/routes/project.py:271]: Yeah true, we already replaced with your implementation of the system_card storage, so we can remove it. But then just for the comment I would not want to run the whole pipeline and waiting for review again if this is the only remark

File: amt/schema/system_card.py (hunk 3 above)
Comment (laurensWe) [reply amt/schema/system_card.py:25]: Yes :) Hence the newly introduced RequirementTask and MeasureTask with a RequirementBase and Requirement and MeasureBase and Measure

**Pull request merged**